api/openapi/specs/api.yaml

```diff
@@ -433,6 +468,45 @@ components:
           additionalProperties:
             $ref: '#/components/schemas/ResourceStats'
           description: A map of resource names to their corresponding statistics
+    InspectHostnames:
+      type: object
+      title: InspectHostnames
+      description: A list of hostnames
+      required: [total, items]
+      properties:
+        total:
+          type: integer
+          example: 200
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/InspectHostname'
+    InspectHostname:
+      type: object
+      title: InspectHostname
+      description: A list of zones in which the hostname is available in
+      required: [hostname, zones]
+      properties:
+        hostname:
+          type: string
+          description: Generated hostname
+          example: redis.redis-system.svc.east.mesh.local
+        zones:
+          type: array
+          items:
+            $ref: '#/components/schemas/InspectHostnameZone'
+    InspectHostnameZone:
+      type: object
+      title: InspectHostnameZone
+      description: A name of the zone with hostname generator that was used to generate the hostname
```

Comment:
I don't understand this sentence.

Reply:
I don't know how to describe a structure with zone and hostname generator name :(
changed with to and, but not sure if it helps. Any suggestions?

Reply:
we talked offline and decided to just remove hostname generator core name from this object